src/wireframes/shapes/neutral/horizontal-scrollbar.ts

```diff
@@ -0,0 +1,107 @@
+import { Rect2 } from '@app/core';
+
+import {
+    Configurable,
+    DiagramItem,
+    SizeConstraint,
+    SliderConfigurable
+} from '@app/wireframes/model';
+
+import { AbstractContext, AbstractControl } from '@app/wireframes/shapes/utils/abstract-control';
+
+import { CommonTheme } from './_theme';
+
+const BAR_SIZE = 'BAR_SIZE';
+const BAR_POSITION = 'BAR_POSITION';
+
+const DEFAULT_APPEARANCE = {};
+DEFAULT_APPEARANCE[DiagramItem.APPEARANCE_FOREGROUND_COLOR] = CommonTheme.CONTROL_TEXT_COLOR;
+DEFAULT_APPEARANCE[DiagramItem.APPEARANCE_BACKGROUND_COLOR] = CommonTheme.CONTROL_BACKGROUND_COLOR;
+DEFAULT_APPEARANCE[DiagramItem.APPEARANCE_STROKE_COLOR] = CommonTheme.CONTROL_BACKGROUND_COLOR;
+DEFAULT_APPEARANCE[DiagramItem.APPEARANCE_STROKE_THICKNESS] = 2;
+DEFAULT_APPEARANCE[DiagramItem.APPEARANCE_TEXT_DISABLED] = true;
+DEFAULT_APPEARANCE[BAR_SIZE] = 50;
+DEFAULT_APPEARANCE[BAR_POSITION] = 0;
+
+const CONFIGURABLES: Configurable[] = [
+    new SliderConfigurable(BAR_SIZE, 'Bar Size'),
+    new SliderConfigurable(BAR_POSITION, 'Bar Position')
+];
+
+const HEIGHT_TOTAL = 20;
+const HEIGHT_CONSTRAINT = new SizeConstraint(undefined, HEIGHT_TOTAL);
```

Comment:
Why do we need this constraint? I would like to change the height of the scrollbar

Reply:
Oh yes. I wasn't sure about this one. I noticed some of the long shapes like Progress have a size constraint so thought maybe this should follow that pattern. But I agree being able to change the height is useful :)